docs/docs/10.10-shallow-compare.md

```diff
@@ -26,7 +26,7 @@ export class SampleComponent extends React.Component {
 ```
 
 `shallowCompare` performs a shallow equality check on the current `props` and `nextProps` objects as well as the current `state` and `nextState` objects.  
-It does this by iterating on the keys of the objects being compared and returning false when the value of a key in each object are not strictly equal.
+It does this by iterating on the keys of the objects being compared and returning true when the value of a key in each object are not strictly equal.
```

Comment:
While we're here, can you also change this to "values of a key in each object are" instead of "value of a key in each object are".  The 'values' token must be plural (one value from each of the two objects) to match plural 'are'.